runtime/marshal.js

```diff
@@ -275,6 +277,8 @@ function caml_input_value_from_reader(reader, ofs) {
           caml_failwith ("input_value: code pointer");
           break;
         case 0x12: //cst.CODE_CUSTOM:
+        case 0x18: //cst.CODE_CUSTOM_LEN:
```

Comment:
Can we handle `CODE_CUSTOM_LEN` separately ? or add a check in the `_j`, `_i`, `_n` cases  to make sure we are not processing a `CODE_CUSTOM_LEN`